### PR TITLE
[docs] Remove ES6 syntax from Gulpfile. Fix link to website in CLI (fixes #780)

### LINKED ISSUE #780: Update website URL in CLI help
https://github.com/DevExpress/testcafe/blob/master/src/cli/argument-parser.js#L35

### PR DESCRIPTION
**Pull request by inikulin**
\cc @VasilyStrelyaev

### CODE REVIEW

File: src/cli/argument-parser.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const DESCRIPTION = dedent(`
 
     You can use one or more file paths or glob patterns to specify which tests to run.
 
-    More info: http://testcafe.devexpress.com/Documentation/CLI
+    More info: https://devexpress.github.io/testcafe/documentation
```
Comment (VasilyStrelyaev) [inline src/cli/argument-parser.js]: What about more a specific link
https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html ?
Or you think it should be memorable?

**Comment by VasilyStrelyaev** [comment]
\r-

File: src/cli/argument-parser.js (hunk 1 above)
Comment (inikulin) [inline src/cli/argument-parser.js]: It's too long, will not fit into the single line of terminal

**Comment by VasilyStrelyaev** [comment]
lgtm,
but I wonder how we can update the documentation tests to not only lint the Gulpfile but also check that we don't add anything wrong to it.

maybe add a dummy gulp task and execute it with the oldest supported node? does it make sense? will it help?

**Pull request merged**